packages/terra-date-picker/src/react-datepicker/month.jsx

```diff
@@ -117,14 +121,14 @@ export default class Month extends React.Component {
 
   getClassNames = () => {
     const { selectingDate, selectsStart, selectsEnd } = this.props
-    return classnames('react-datepicker__month', {
-      'react-datepicker__month--selecting-range': selectingDate && (selectsStart || selectsEnd)
+    return classnames('react-datepicker-month', {
```

Comment:
This could be changed to an array

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/794/commits/81ee9abb34be2bdf4ab2fe925946613120f06702.